### DISCUSSION TITLE
Is this mod safe????

### DISCUSSION THREAD

**Discussion by froginalog** [mega12345mega/NBT-Editor#128] [General]
![image](https://github.com/user-attachments/assets/1bb29d59-e9f0-4ae5-a768-c87d3ae02e51)

**Comment by mega12345mega** [comment]
Where did you download your version from? The mod doesn't do either of those things. The official download is [https://modrinth.com/mod/nbt-editor](https://modrinth.com/mod/nbt-editor).
If you are unsure, you can always read the source code and build the mod yourself.

Edit: I didn't see the Discord messages before sending this, but if you have any questions still feel free to ask
